### ISSUE TITLE
Resources are not merged together

### ISSUE THREAD

**Issue by LoZeno** [icarus-sullivan/serverless-plugin-modularize#2]
Despite what the Readme says, the plugin does not seem to actually merge the resources section of the modules:

have hello.module.yml define a resource section with a DynamoDB table
```
resources: 
  Resources:
      usersTable:
        Type: AWS::DynamoDB::Table
        Properties:
          TableName: usersTable
          AttributeDefinitions:
            - AttributeName: email
              AttributeType: S
          KeySchema:
            - AttributeName: email
              KeyType: HASH
          ProvisionedThroughput:
            ReadCapacityUnits: 1
            WriteCapacityUnits: 1
```

have goodbye.module.yml define a resource section with another DynamoDB table:
```
resources: 
  Resources:
      carsTable:
        Type: AWS::DynamoDB::Table
        Properties:
          TableName: carsTable
          AttributeDefinitions:
            - AttributeName: registration
              AttributeType: S
          KeySchema:
            - AttributeName: registration
              KeyType: HASH
          ProvisionedThroughput:
            ReadCapacityUnits: 1
            WriteCapacityUnits: 1
```

then run the serverless deploy: the stack will contain only the resources of the last module loaded.

**Comment by LoZeno** [comment]
I think the problem is in this line:
https://github.com/icarus-sullivan/serverless-plugin-modularize/blob/4a2f3fa12c041722848c1b77408ce4f3c073548d/src/utils/merge.js#L17

the `Resources:` property gets overwritten, instead of merged, so only the last module loaded will be merged in (as it's the module that will overwrite last).

**Comment by icarus-sullivan** [comment]
If you running latest, I will take a look at it soon. I know there was some merge issues with early versions. It might be related. Apologies for my lateness, I was on vacation before starting a new job.

**Comment by icarus-sullivan** [comment]
@LoZeno Appreciate you reaching out, I mentioned you in the Changelog. The issue should be fixed with v1.0.6. 

https://www.npmjs.com/package/serverless-plugin-modularize
